full-stack-tests/ecschema-rpc-interface/src/frontend/setup/IModelSession.ts

```diff
@@ -24,7 +23,7 @@ export class IModelSession {
 
   public async open(): Promise<CheckpointConnection> {
     try {
-      const env = Config.App.get("imjs_buddi_resolve_url_using_region");
+      const env = process.env.imjs_buddi_resolve_url_using_region;
```

Comment:
what happens when you use `process.env` on the frontend? Does it get resolved by webpack? Is that what we want here?

Reply:
Yes, as I understand that is what happens - webpack will resolve it at build time. @wgoehrig can correct me if wrong.

Reply:
Yes, that's the correct way it works for these tests so anytime they change you need to re-build.  @kabentley going to resolve this but we can re-open if you have further concerns.